go/src/koding/klient/os/environ.go

```diff
@@ -0,0 +1,78 @@
+package os
+
+import (
+	"errors"
+	"fmt"
+	"sort"
+	"strings"
+)
+
+// Environ is a convenience wrapper for environment variables
+// produced by os.Environ.
+type Environ map[string]string
+
+func (e Environ) Encode(m Environ) []string {
```

Comment:
This function is rather MergeAndEncode. Can we create e.Encode() []string and e.Merge(Environ) Environ ?